frontend/src/bundles/talent-onboarding/components/skills-step/components/skills-project-links.tsx

```diff
@@ -66,6 +74,12 @@ const SkillsProjectLinks: React.FC<Properties> = ({ control, name }) => {
                             name={fieldPath}
                             {...withoutReferenceAndName}
                         />
+                        {index !== 0 && (
+                            <CloseIconButton
+                                index={index}
+                                onClick={removeLink}
+                            />
+                        )}
```

Comment:
When link is long closeIcon is overlap with it. Also if there is an error this Icon also move from it`s place
![Screenshot_4](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/67747035/3d64aa17-2169-49f1-9f90-6663f91b85b7)
I am not sure about it, but even link without www/http/https is treated as valid (link.com). I think we need to clarify it with qa team

Reply:
@MChornomaz what about this comment?

Reply:
Fixed